WordPress/src/main/java/org/wordpress/android/util/config/WordPressInAppUpdateBlockingVersionConfig.kt

```diff
@@ -0,0 +1,18 @@
+package org.wordpress.android.util.config
+
+import org.wordpress.android.annotation.RemoteFieldDefaultGenerater
+import javax.inject.Inject
+
+const val WP_IN_APP_UPDATE_BLOCKING_VERSION_REMOTE_FIELD = "wp_in_app_update_blocking_version_android"
+const val WP_IN_APP_UPDATE_BLOCKING_VERSION_DEFAULT = "0"
+
+@RemoteFieldDefaultGenerater(
+    remoteField = WP_IN_APP_UPDATE_BLOCKING_VERSION_REMOTE_FIELD,
+    defaultValue = WP_IN_APP_UPDATE_BLOCKING_VERSION_DEFAULT
+)
+
+class WordPressInAppUpdateBlockingVersionConfig @Inject constructor(appConfig: AppConfig) :
+    RemoteConfigField<Int>(
+        appConfig,
+        WP_IN_APP_UPDATE_BLOCKING_VERSION_REMOTE_FIELD
+    )
```

Comment:
Hey Pantelis 👋🏼 
[Minor Suggestion]: I can see two remote config fields, one for WP and One for JP.  Can we create the same class under the Wordpress and Jetpack package with one name, `InAppUpdateBlockingVersionConfig`? This will simplify that usage, I feel. No need for and if-else check depending on the current app. 

An example of what I have in mind is `LoginPrologueRevampedFragment`, which is under the Jetpack and Wordpress packages.

Reply:
Thanks @AjeshRPai ! I will take care of this!